src/python/.gitignore

```diff
@@ -0,0 +1 @@
+gen/
```

Comment:
The C++ proto output directory is gens/, I figured I would be consistent with that.